tests/experimental/test_experimental_base.py

```diff
@@ -17,10 +18,8 @@ class DummyModel(PyTorchBackend):
         A dummy implementation of PyTorchBackend for testing purposes.
         """
 
-        def __init__(self, x=None, y=None, random_seed=None, **kwargs):
+        def __init__(self, x=None, y=None, **kwargs):
             super().__init__()
-            if random_seed is not None:
-                self.set_random_seed(random_seed)
             _, _ = x, y  # unused variables
             self.linear = nn.Linear(1, 1)
```

Comment:
Perhaps it would be worth retaining this for subclass API so that there remains an option (though not the default) to set the random seed as part of the init?

Reply:
Did you mean for all Emulators or a particular subclass? The line you've commented here is in the dummy implementation of PyTorchBackend

Reply:
I think it makes sense to be part of a subclass specific model API (but with default typically `seed = None`) since not all models have any randomness (the dummy impl is the only example at the moment but more to be added soon and this would be an example impl) - what do you think?

Reply:
Yeah I'm with you, have updated that, so for the models we have:
- LGBM and RF have the random_seed arg, but is passed into underlying model so this line isn't needed
- This dummy example (and future implemented models) we have this in the model init:
    ```
        if random_seed is not None:
                set_random_seed(random_seed)
    ```
- Other models do not have random_seed arg